### DISCUSSION TITLE
Hubs cloud Digital Ocean hosting

### DISCUSSION THREAD

**Discussion by Letitbe133** [mozilla/hubs#2941] [Hubs Cloud Questions] [answered]
Hi there 🖖. Just saw that it was possible to create a hubs cloud instance on Digital Ocean. Anybody did this already and could give a feedback? Sorry if question's been asked already ☺

**Comment by emclaren** [comment, marked as answer]
Hi @Letitbe133 - thanks for asking this. AWS is currently the most supported path. Digital Ocean is more of a  "beta" right now. Which you should use depends on your needs. Digital Ocean can only run on a single instance, where AWS supports multiple servers working together. This could be important if you plan to have a lot of concurrent users.

I see that you are also in our Discord Community (https://discord.gg/wHmY4nd) - I believe you asked about this there as well. I just re-posted your question to see if there was any community feedback. I definitely recommend also searching for "Digital Ocean" in the Discord, as I know some folks have talked about setting it up before.

**Reply by emclaren** [reply]
Also - if you have any questions about the setup process, feel free to start a new discussion thread for them. Folks on our team should be able to provide answers if you have something specific about the Digital Ocean set up you are looking to learn.

**Reply by robinkwilson** [reply]
Here's another link about SMTP settings for DO deploys: https://github.com/mozilla/hubs/discussions/2886

**Reply by Letitbe133** [reply]
Hi @emclaren @robin-k-wilson and thx for your reply :slightly_smiling_face:  
Right now we're trying to build a local community using Discord and Hubs and we were wondering maybe hosting our own Hubs Cloud instance would be cool and more effective performance wise. We would only need 1 instance for our use case so it's fine. I'm gonna search Discord and see what infos I can get ;)
Just need to figure out what would be the benefits of hosting our own Hubs Cloud instance and what would it cost
